hw/vendor/patches/pulp_riscv_dbg/0003-Fix-lint.patch

```diff
@@ -0,0 +1,45 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
There is already a lint fix patch. I think the new fixes of this patch could have been merged into the other patch. But in the interest of time, I think this is okay.

Reply:
Hm since we actually have a patch repository we should move the patches there if possible.
This will make it easier to modify&replay these patches in the future.

Sorry I don't want to unnecessarily draw this out - let me know if I can help with this.